packages/dart/noports_core/lib/src/sshnp/impl/sshnp_openssh_local_impl.dart

```diff
@@ -93,6 +89,11 @@ class SshnpOpensshLocalImpl extends SshnpCore
       );
     }
 
+    /// Find a port to use
+    final server = await ServerSocket.bind(InternetAddress.anyIPv4, 0);
+    int localRvPort = server.port;
+    await server.close();
```

Comment:
need to do this before we spawn the process, because we need to know what port the srv is using